File: Week 1 C - Basics/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("How tall is the pyramid? \n");
        if (height <= 0)
        {
            printf("You must introduce a whole number and a value greater than 0. \n");
        }
    }
    while (height <= 0);

    for (int row = 0; row < height; row++)
    {
        for (int col = 0; col < height; col++)
        {
            if (col < height - row - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: Week 6 Python/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    pyramyd_height = get_int("How tall is the pyramid? ")
    if pyramyd_height >= 1 and pyramyd_height <= 8:
        break
    print("You must introduce a number between 1 and 8")


for row in range(pyramyd_height):
    for col in range(pyramyd_height - row - 1):
        print(" ", end="")
    for col in range(row + 1):
        print("#", end="")
    print("  ", end="")
    for col in range(row + 1):
        print("#", end="")
    print()

